main.cc

```diff
@@ -1740,30 +1740,42 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
   }
 }
 
+using main_func_type = std::function<int(int, char**)>;
+static main_func_type lookup_main_func(std::string_view name) {
+    std::pair<std::string_view, main_func_type> funcs[] = {
+        {"server", scylla_main},
+        {"types", tools::scylla_types_main},
+        {"sstable", tools::scylla_sstable_main},
+    };
+    auto found = std::ranges::find_if(funcs, [name](auto& name_and_func) {
```

Comment:
@avikivity can we now use ranges?

Reply:
Please add space between `]` and `(`.

Reply:
@denesb hi Botond, we are already using them =)
```console
$ git grep std::ranges | wc -l
45
```
BTW, i recall that Avi asked the same question when i was using ranges in Seastar!

Reply:
I once started sending patches converting us from `boost::ranges` to `std::ranges`, then we had to revert these due to some bugs in the standard library. Maybe these bugs were fixed, but every time I see `std::range` I think of this.

Reply:
i see. that's sad. probably the new gcc in the toolchain brought a more C++20 compliant libstdc++ to us.

Reply:
The bug might have been fixed since.